src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Authentication/AuthenticationService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Http;
+using SFA.DAS.CommitmentsV2.Authentication;
+using SFA.DAS.CommitmentsV2.Types;
+using IAuthenticationService = SFA.DAS.CommitmentsV2.Authentication.IAuthenticationService;
+
+namespace SFA.DAS.CommitmentsV2.Api.Authentication
+{
+    public class AuthenticationService : IAuthenticationService
+    {
+        private readonly IHttpContextAccessor _httpContextAccessor;
+
+        public AuthenticationService(IHttpContextAccessor httpContextAccessor)
+        {
+            _httpContextAccessor = httpContextAccessor;
+        }
+
+        public bool IsUserInRole(string role)
+        {
+            return _httpContextAccessor.HttpContext.User.IsInRole(role);
+        }
+
+        public Originator GetUserRole()
```

Comment:
Possibly better to leave it until the two branches are merged to the combined branch otherwise I'm going to be duplicating the set up and increasing the merge conflicts.

Reply:
Added UT as code from Elliot's branch (from whence this came) has now been removed